src/Symfony/Component/HttpKernel/Kernel.php

```diff
@@ -469,6 +463,32 @@ protected function initializeBundles()
     }
 
     /**
+     * @param BundleInterface $bundle
+     * @param bool $dependency
+     */
+    protected function addBundle($bundle, $dependency = false)
+    {
+        $name = $bundle->getName();
+
+        if (isset($this->bundles[$name]) && false === $dependency) {
```

Comment:
I don't understand what you want :/
You mean that if I have two bundles with the same name, the latter won't be included at all ? If it's this behavior you're talking about, it's wrong IMO. The "duplicated" bundles must be warned by the developer, as a new bundle is almost all the time added in `dev` environment, so the developer can surely view the exception message and fix its bug. Plus, the bundles order would have a significant role, and it is not an expected behavior at all.